lib/ansible/template/__init__.py

```diff
@@ -109,6 +109,7 @@ def generate_ansible_template_vars(path, dest_path=None):
     temp_vars = {
         'template_host': to_text(os.uname()[1]),
         'template_path': path,
+        'template_path_relative': re.sub(os.getcwd(), '', path),
```

Comment:
this is incorrect, as we dont use os.getcwd() to find relative files, see the 'dwim' functions.